spec/index.html

```diff
@@ -1152,6 +1152,23 @@ <h3>The XML Schema Built-in Datatypes</h3>
         model.</li>
     </ul>
 
+    <p class="note">As mentioned in the table above, the <a>value spaces</a>
```

Comment:
The definition of value space is given by https://www.w3.org/TR/xmlschema11-2/#sec-vs-double and https://www.w3.org/TR/xmlschema11-2/#sec-vs-float. 

It says "The double datatype is patterned after the IEEE double-precision 64-bit floating point datatype" so it is not claiming to be exactly IEEE 754-2008.